*** esdm-uncontoured.mml	2018-12-05 19:56:55.717147673 +0000
--- esdm-uncontoured.mml	2018-12-05 19:59:28.918674800 +0000
***************
*** 1,7 ****
  scale: 1
  metatile: 2
  name: OpenStreetMap Carto
! description: A general-purpose OpenStreetMap mapnik style, in CartoCSS
  bounds: &world
    - -180
    - -85.05112877980659
--- 1,7 ----
  scale: 1
  metatile: 2
  name: OpenStreetMap Carto
! description: An ESDM OpenStreetMap mapnik style, in CartoCSS
  bounds: &world
    - -180
    - -85.05112877980659
***************
*** 14,20 ****
  format: png
  interactivity: false
  minzoom: 0
! maxzoom: 22
  srs: "+proj=merc +a=6378137 +b=6378137 +lat_ts=0.0 +lon_0=0.0 +x_0=0.0 +y_0=0.0 +k=1.0 +units=m +nadgrids=@null +wktext +no_defs +over"
  
  # Various parts to be included later on
--- 14,20 ----
  format: png
  interactivity: false
  minzoom: 0
! maxzoom: 18
  srs: "+proj=merc +a=6378137 +b=6378137 +lat_ts=0.0 +lon_0=0.0 +x_0=0.0 +y_0=0.0 +k=1.0 +units=m +nadgrids=@null +wktext +no_defs +over"
  
  # Various parts to be included later on
***************
*** 36,49 ****
      extent: "-20037508,-20037508,20037508,20037508"
  
  Stylesheet:
!   - style.mss
!   - fonts.mss
    - shapefiles.mss
    - landcover.mss
!   - water.mss
    - water-features.mss
!   - road-colors-generated.mss
!   - roads.mss
    - power.mss
    - placenames.mss
    - buildings.mss
--- 36,49 ----
      extent: "-20037508,-20037508,20037508,20037508"
  
  Stylesheet:
!   - esdm/style-OS.mss
!   - esdm/fonts.mss
    - shapefiles.mss
    - landcover.mss
!   - esdm/water-OS.mss
    - water-features.mss
!   - esdm/road-colors-OS.mss
!   - esdm/roads-OS.mss
    - power.mss
    - placenames.mss
    - buildings.mss
